avocado/core/nrunner/runnable.py

```diff
@@ -455,10 +447,9 @@ def pick_runner_command(kind, runners_registry=None):
         # runner convention within the avocado.plugins.runners namespace dir.
         # Looking for the file only avoids an attempt to load the module
         # and should be a lot faster
-        module_name = kind.replace("-", "_")
```

Comment:
With these changes it came a question in my mind which is not totally related to this PR. What about moving the `pick_runner_command` method and all the related methods to their own utility in `avocado.core.utils`. IMO these methods are not related to the Runnable, only to the runnable kind and I don't see a reason why they should be part of Runnable class. What do you think about this?

Reply:
Yep, makes sense to me.  I'll try to some refactors on this soon and will take this suggestion into account.